includes/class-mailer.php

```diff
@@ -87,7 +87,11 @@ public static function comment_notification_text( $message, $comment_id ) {
 		}
 
 		$post                  = \get_post( $comment->comment_post_ID );
-		$comment_author_domain = \gethostbyaddr( $comment->comment_author_IP );
+		$comment_author_domain = '';
+		// Only attempt to resolve hostname if we have a valid IP address.
```

Comment:
```suggestion

		// Only attempt to resolve hostname if we have a valid IP address.
```
I'm a big fan of setting defaults and conditionally changing them, and doing that in one block to make it easy to see they belong together!

Unless there's an inline comment, where making it easy to catch takes precedence for me? Let's give comments room to breath